public/style.css

```diff
@@ -25,11 +25,6 @@ li {
   margin: auto;
 }
 
-#user .home-label {
-  background: linear-gradient(#000,  #00b0ff);
-  border: 2px solid  #00b0ff;
-}
-
```

Comment:
@kots14 this is what I am talking about for jumbotron, you can add this code in your next commit.

Reply:
got it.